xt/66-cucumber/11-ar/step_definitions/pageobject_steps.pl

```diff
@@ -19,9 +19,6 @@
 use Test::BDD::Cucumber::StepFile;
 
 
-use Data::Dumper;
-
-
```

Comment:
Even though Data::Dumper should probably not be there, it should be PR-ed in a separate PR. (11-ar has really nothing to do with this PR...)